### PR TITLE
Firsthm

### PR DESCRIPTION
**Pull request by Daniluko**

### CODE REVIEW

File: src/main/java/springboot/controller/CustomerController.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import java.util.Objects;
+
+/**
+ * Created by Danylo on 31.03.2019
+ */
+@RestController
+@RequestMapping("/customers")
+public class CustomerController {
+    private static final Logger LOG = LogManager.getLogger(CustomerController.class);
+
```
Comment (Artsapollo) [inline src/main/java/springboot/controller/CustomerController.java:22]: Don't forget about getAllCustomers method

File: src/main/java/springboot/controller/CustomerController.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    @PostMapping
+    public void insertCustomer(@Valid @RequestBody Customer customerRequest) {
+        LOG.info("Insert customer, customerRequest={}", customerRequest);
+        customerService.insertCustomer(customerRequest);
+        LOG.info("successfully");
+    }
+
+    @PutMapping("/{id}")
+    public void updateCustomerById(@PathVariable("id") BigDecimal id, @RequestParam("cust_num") BigDecimal cust_num){
```
Comment (Artsapollo) [inline src/main/java/springboot/controller/CustomerController.java]: Think about to update not customer id as cust_num but something else

File: src/main/java/springboot/entity/Customer.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+@Entity
+@Table(name = "CUSTOMERS", schema = "MA_STUDENT")
+public class Customer implements java.io.Serializable {
+
+    @Id
+    @Column(name = "CUST_NUM")
+    private BigDecimal custNum;
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "CUST_REP")
+    private Salesrep custRep;
```
Comment (Artsapollo) [inline src/main/java/springboot/entity/Customer.java:19]: Don't forget @JsonIgnore

**Review by Artsapollo — APPROVED** [review]
Nice! Read comments

**Review by SerhiiPasko — APPROVED** [review]
Good

**Pull request merged**